src/virtual_secondary/partialverificationsecondary.cc

```diff
@@ -37,14 +37,15 @@ data::InstallationResult PartialVerificationSecondary::putMetadata(const Target
   }
 
   // TODO(OTA-2484): check for expiration and version downgrade
-  root_ = Uptane::Root(RepositoryType::Director(), Utils::parseJSON(director_root), root_);
+  Uptane::Root root(Root::Policy::kAcceptAll);
+  root = Uptane::Root(RepositoryType::Director(), Utils::parseJSON(director_root), root);
```

Comment:
This is a downgrade in functionality by not storing the Root to check it the next time around. However, this whole class is woefully incomplete, so it doesn't really matter. It needs a lot of work and that isn't part of the scope here.